src/AppInstallerCLICore/Workflows/PortableFlow.cpp

```diff
@@ -428,10 +428,10 @@ namespace AppInstaller::CLI::Workflow
                 AICLI_LOG(CLI, Info, << "Unable to create symlink. '" << symlinkFullPath << "points to an existing directory.");
                 AICLI_TERMINATE_CONTEXT(APPINSTALLER_CLI_ERROR_PORTABLE_SYMLINK_PATH_IS_DIRECTORY);
             }
-            else
+            else if (std::filesystem::remove(symlinkFullPath))

```

Comment:
I assume that `create_symlink` doesn't allow overwriting an existing symlink?  It isn't clear what will happen from either location I looked at, but I would guess that it doesn't.

Reply:
Yes that's right. If we don't remove the existing file, `create_symlink` will fail and throw and error that the "file already exists"